ksqldb-examples/src/main/java/io/confluent/ksql/datagen/DataGen.java

```diff
@@ -229,7 +233,7 @@ private static void usage() {
               .put("value-format", (builder, arg) -> builder.valueFormat = parseFormat(arg))
               // "format" is maintained for backwards compatibility, but should be removed later.
               .put("format", (builder, argVal) -> builder.valueFormat = parseFormat(argVal))
-              .put("value_delimiter",
+              .put("valueDelimiter",
```

Comment:
A note: The config is already weird in that it uses both snake case and camel case for arg names :(

So I had to choose one. I chose camel case because more properties seem to use that than snake case.